rest_api/test/test_rest_api.py

```diff
@@ -265,22 +233,24 @@ def test_file_upload(client: TestClient, api_document_store: BaseDocumentStore):
     response = client.post(url="/file-upload", files=file_to_upload, data={"meta": '{"test_key": "test_value"}'})
     assert 200 == response.status_code
 
-    documents = api_document_store.get_all_documents()
-    assert len(documents) > 0
-    for doc in documents:
-        assert doc.meta["name"] == "sample_pdf_1.pdf"
-        assert doc.meta["test_key"] == "test_value"
+    # ensure the converter was called with the right params
+    _, kwargs = MockPDFToTextConverter.mocker.call_args
+    # Files will be converted to something like 83f4c1f5b2bd43f2af35923b9408076b_sample_pdf_1.pdf
+    # so we ensure the original file name is contained in the converted file name
+    assert "sample_pdf_1.pdf" in kwargs["file_path"].parts[-1]
```

Comment:
What is `parts()` doing? At a quick skim of the Python documentation I could not find mention of a method with this name.

Reply:
https://docs.python.org/3/library/pathlib.html#accessing-individual-parts

Reply:
Ooh right :smile: I assumed the file path was a string.